meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/apalis-imx6/lmp-secure.cfg

```diff
@@ -0,0 +1 @@
+CONFIG_TDX_CFG_BLOCK_DEV=2
```

Comment:
Might be better here to duplicate lmp.cfg used by apalis-imx6 mfgtools, and just make it available under apalis-imx6-sec, otherwise I wonder if this will later get confusing as we will be handling multiple config fragments.